### DISCUSSION TITLE
isolated instance

### DISCUSSION THREAD

**Discussion by sclerometer** [mastodon/mastodon#17931] [Q&A]
is there any way how to make isolated mastodon instance without federation?

our support group is looking for a service, where we can share our thoughts in safe, private space without it leaking on the internet. now we use mybb forum behind vpn, but we are looking for something more twitter or thumblr like.

**Comment by ClearlyClaire** [comment]
Looks like what you're looking for for is `LIMITED_FEDERATION_MODE`, see https://docs.joinmastodon.org/admin/config/#limited_federation_mode

**Reply by VityaSchel** [reply]
The problem with this option is that it disables all public pages (logged out users won't be able to browse your site) and profiles directory.

**Reply by StrangeMinds1960** [reply]
I think that's *exactly* what the OP was after
